pnpm-workspace.yaml

```diff
@@ -5,3 +5,4 @@ packages:
   - web
   - proxy
   - loader-tests
+  - extension
```

Comment:
pnpm has great monorepo support (when you have multiple `package.json` in dirs).

By having `pnpm-workspace.yaml` you can specify project’s folders and they all will have common `pnpm-lock.yaml`.